animal_adoption/front_end/src/components/Signup_Form_User.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from "react";
+
+export default class Signup_Form_User extends Component{
```

Comment:
Good job, Ryan! Just one thing about coding styles.
This is a matter of preference and I think there are no official guide, 
but PascalCase is widely used for React component name and camelCase for props, variables in javascript files.
ex) 
component/class name: SigunupFormUser, SignupPage
variable name: firstName, lastName

here is style guides of big tech companies.
https://codeburst.io/5-javascript-style-guides-including-airbnb-github-google-88cbc6b2b7aa

I don't have any preferences but it's good to unify our coding styles.
I think either Airbnb or Google style is good for our development.
So I'm going to setup ESLint to check our coding styles automatically today or tomorrow.
Then, let you know when the setup is ready!